.github/workflows/continuous-integration.yml

```diff
@@ -41,8 +43,19 @@ jobs:
             .cache/yarn
             node_modules
 
+      - name: List changed files
+        id: changed-files
+        run: echo ::set-output name=all_changed_files::$(git diff --name-only origin/master...)
+
+      - name: Get app entries for changed files
+        id: get-changed-apps
+        if: ${{ github.ref != 'refs/heads/master' && matrix.buildtype == 'vagovdev' }}
+        run: echo ::set-output name=app_entries::$(node script/github-actions/get-changed-apps.js)
+        env:
+          CHANGED_FILE_PATHS: ${{ steps.changed-files.outputs.all_changed_files }}
+
       - name: Build
-        run: yarn build --verbose --buildtype=${{ matrix.buildtype }}
+        run: yarn build --verbose --buildtype=${{ matrix.buildtype }} --entry=${{ steps.get-changed-apps.outputs.app_entries }}
```

Comment:
This isn't a problem, but I'm not sure if others would be confused by the empty `entry` option being used for builds other than only changed apps. Sucks there isn't a clean way to do this in GHA. The only other options I can think of are using an `if` statement in bash, or making two separate steps for the two types of builds.

Reply:
That's true, ideally we'd omit the argument completely if it's empty. It looks like that actually is [possible in bash](https://stackoverflow.com/questions/20306739/how-to-not-pass-empty-quoted-variables-as-arguments-to-commands). I'm guessing GHA wouldn't like that syntax though.

But on the other hand, hopefully over time more and more of the builds will be single-app builds anyway. I guess it's a question of whether to make this step more complex in order to make the log output a bit simpler, or to go the other way

Reply:
I got the bash parameter expansion method to work with an environment variable. GHA output with app entries present looks like:
```
Run yarn build --verbose --buildtype=vagovdev ${ENTRY:+"--entry=$ENTRY"}
yarn run v1.22.15
$ node script/prebuild.js
$ ./script/build.sh --scaffold --verbose --buildtype=vagovdev --entry=vaos
```
With no entries it looks like:
```
Run yarn build --verbose --buildtype=vagovprod ${ENTRY:+"--entry=$ENTRY"}
yarn run v1.22.15
$ node script/prebuild.js
$ ./script/build.sh --scaffold --verbose --buildtype=vagovprod
```
I think this is the cleanest way.

Reply:
That's pretty neat! Nice refactoring, I definitely think this is the cleanest way we can do this.  ➕1 for parameter expansion!

Reply:
That was definitely an exciting find! Since it's working, I'll revert the VAOS test and we'll be all set with this PR